### PR TITLE
refactor: fix image path to avoid warning in build

### PR DESCRIPTION
**Pull request by RonenMars**
## Description
Fix this warning:
image

## Linear Ticket
https://linear.app/autokitteh/issue/UI-820/fix-asset-image-warning-in-build

## What type of PR is this? (check all applicable)

-   [ ] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [x] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/components/organisms/dashboard/welcomeMainBlock.tsx [hunk 1]
```diff
@@ -100,7 +100,7 @@ export const DashboardWelcomeMainBlock = () => {
 					}
 					videoStyle={{
 						backgroundColor: "#1d2226",
-						backgroundImage: "url(/assets/image/pages/intro/startingProject.jpg)",
+						backgroundImage: "url(@assets/image/pages/intro/startingProject.jpg)",
```
Comment (J1za) [inline src/components/organisms/dashboard/welcomeMainBlock.tsx:103]: I don't see image with `@`

File: src/components/organisms/dashboard/welcomeMainBlock.tsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ export const DashboardWelcomeMainBlock = () => {
 		<div className="z-10 mt-7 grid select-none gap-5">
 			<div className="col-span-1 grid grid-cols-auto-fit-350 items-stretch gap-4 overflow-visible rounded-2xl border border-gray-950 bg-black p-8 pl-6 pr-4 font-averta text-white">
 				<div className="flex min-h-52 flex-col">
-					<div className="flex w-full flex-1 items-center justify-center rounded-2xl border-2 border-gray-750 bg-[#1d2226] bg-[url('/assets/image/pages/intro/main.jpg')] bg-contain bg-center bg-no-repeat">
+					<div className="flex w-full flex-1 items-center justify-center rounded-2xl border-2 border-gray-750 bg-[#1d2226] bg-[url('@assets/image/pages/intro/main.jpg')] bg-contain bg-center bg-no-repeat">
```
Comment (J1za) [inline src/components/organisms/dashboard/welcomeMainBlock.tsx:37]: so, also add `@`  for IntroMainBlock component

Comment (RonenMars) [reply src/components/organisms/dashboard/welcomeMainBlock.tsx]: refactored

File: src/components/organisms/dashboard/welcomeMainBlock.tsx (hunk 1 above)
Comment (RonenMars) [reply src/components/organisms/dashboard/welcomeMainBlock.tsx]: refactored

**Review by J1za — APPROVED** [review]

**Pull request merged**